docs/ContentTypeParser.md

```diff
@@ -24,14 +24,29 @@ You can also use the `hasContentTypeParser` API to find if a specific content ty
 ```js
 if (!fastify.hasContentTypeParser('application/jsoff')){
   fastify.addContentTypeParser('application/jsoff', function (req, done) {
-    //code to parse request body /payload for given content type 
+    //code to parse request body /payload for given content type
   })
 }
 ```
 
+#### Body Parser
+You can parse the body of the request in two ways, the first one is showed above, you just add a custom content type parser and handle the request stream. In the second one you should pass a `parseAs` option to the `addContentTypeParser` API, where you declare how you want to get the body, it could be `'string'` or `'buffer'`. If you use the `parseAs` option Fastify will internally handle the stream and perform some checks, such as the [maximum size](https://github.com/fastify/fastify/blob/master/docs/Factory.md#factory-body-limit) of the body and the content length.
```

Comment:
"You can parse the body of the request in two ways, the first one is showed above, you just add a custom content type parser and handle the request stream." => "You can parse the body of the request in two ways. The first one is shown above: you add a custom content type parser and handle the request stream."